### DISCUSSION TITLE
[Errno 28] No space left on device

### DISCUSSION THREAD

**Discussion by cristinarumi** [Parcels-code/Parcels#1621] [General]
Hi,

I've encountered the "[Errno 28] No space left on device" error while running OceanParcels, indicating insufficient disk space. However, I have over 14 TB of free space available.
<img width="742" alt="Screenshot 2024-07-27 at 10 18 22" src="https://github.com/user-attachments/assets/b1a49ea2-0cde-4e75-bc2d-3319330eb7dc">

Do you have any suggestions on how to resolve this issue? I would really appreciate your help.

Thank you,
Cristina

**Comment by VeckoTheGecko** [comment]
Hi @cristinarumi ,
We have never encountered this issue while using parcels, and looking at the trace its an error that is raised by `zarr` (one of our dependencies).

I think this problem is specific to your machine and not a code issue. I would recommend discussing with your system administrator to help further, and make sure that space is available to your simulation to be written into.

Unfortunately there's not much more we can do here to support working with this information.
